src/os/CMakeLists.txt

```diff
@@ -128,3 +129,29 @@ endif()
 if(WITH_LTTNG AND WITH_EVENTTRACE)
   add_dependencies(os eventtrace_tp)
 endif()
+
+if(WITH_LIBURING)
+  include(ExternalProject)
+  if("${CMAKE_GENERATOR}" MATCHES "Make")
+    set(make_cmd "$(MAKE)")
+  else()
+    set(make_cmd "make")
+  endif()
```

Comment:
@rouming i take the liberty to define `make_cmd` instead of hardwiring it to `$(MAKE)` which only works if the cmake generator happens to be "make" as well.  if one is using other generators like scons or ninja `$(MAKE)` won't be expanded by their interpreters.